src/Sarif.Converters/FxCopConverter.cs

```diff
@@ -119,10 +148,7 @@ internal Result CreateResult(FxCopLogReader.Context context)
                 !string.IsNullOrWhiteSpace(targetFile) &&
                 !sourceFile.Equals(targetFile))
             {
-                result.AnalysisTarget = new FileLocation()
-                {
-                    Uri = new Uri(targetFile, UriKind.RelativeOrAbsolute)
-                };
+                result.AnalysisTarget = BuildFileLocationFromFxCopReference(targetFile);
```

Comment:
>BuildFileLocationFromFxCopReference [](start = 40, length = 35)

We could use a naming convention for methods like this. In FileInfoFactory.cs, where you had a method that both created a `FileInfo` object and added it to the `Files` dictionary, I suggested you factor out a creation method `private FileData CreateFrom(PhysicalLocation)`. I'm not wedded to that convention at all, it's just that this is the first time I'm seen a creation method in this code base named `BuildXxx`.